package.json

```diff
@@ -26,18 +26,14 @@
     "url": "https://github.com/learn-co-curriculum/js-data-structures-objects-lab/issues"
   },
   "homepage": "https://github.com/learn-co-curriculum/js-data-structures-objects-lab#readme",
-  "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-preset-env": "1.7.0",
+  "dependencies": {
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
     "chai": "^4.2.0",
     "file-system": "^2.2.2",
-    "jsdom": "^9.2.1",
+    "jsdom": "^15.1.1",
```

Comment:
6️⃣ ❗️

Reply:
@rrcobb Confirmed. Tested spread solutions against this harness. Will merge to `solution` in another PR